irohad/torii/command_service.hpp

```diff
@@ -52,7 +52,7 @@ namespace torii {
      */
     virtual rxcpp::observable<
         std::shared_ptr<shared_model::interface::TransactionResponse>>
-    getStatusStream(const shared_model::crypto::Hash &hash) = 0;
```

Comment:
Please revert this change, as it can be fixed without modifications in the interface.